### PR TITLE
I'm Done!

### PR DESCRIPTION
**Pull request by webbedfooty**

### CODE REVIEW

File: lib/strings.md [hunk 1]
```diff
@@ -5,20 +5,27 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+A string is a sequence of characters, that are, wait for it, strung together; identified by either single or double quotation marks. 
 
 # What are some examples of information that would be Strings as opposed to some other data type?
 
-Replace me with your response...
+Anything to be processed simply as text, as opposed to numeric or Boolean values which are evaluated/analyzed.
```
Comment (andrewek) [inline lib/strings.md]: Nice.

File: lib/strings.md [hunk 2]
```diff
@@ -5,20 +5,27 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
...
 
 # What is one way, using Ruby, to retrieve the 6th character in a String like `"Ada Lovelace"`? How about the 8th character? What happens if you try to retrieve the value of the _99th_ character (Or any character that doesn't exist)?
 
-Replace me with your response...
+A = "Ava Lovelace"
```
Comment (andrewek) [inline lib/strings.md]: A few notes:

1. names in Ruby, unless they are names of classes (e.g. String, Num, etc.), should start with lower-case letters. Thus, you should have `a = "Ava Lovelace"`
2. The `byteslice` method is fine, but there's a more terse way to do it. What if you do `a[ 7 ]`? Does that give you the same thing as `a.byteslice(7)`?

**Comment by andrewek** [comment]
@webbedfooty See my note about byteslices and update appropriately. Then you can re-commit, re-push, and the pull request will update itself. Good stuff!

**Comment by webbedfooty** [comment]
I did the "git add" process in Cloud 9 yesterday abut I don't see the comments add here
Am I supposed to update my comments here?
If so....

---   Epilogue  ---

   A few notes:
#1	names in Ruby, unless they are names of classes (e.g. String, Num, etc.), should start with lower-case letters. Thus, you should have a = "Ava Lovelace" 

yes absolutely - totally overlooked that but understand the need for them to start with lower-case

#2	The byteslice method is fine, but there's a more terse way to do it. What if you do a[ 7 ]? Does that give you the same thing as a.byteslice(7)?

Yes, you do get the same thing using either method. 
However, with the number set at 7, it actually gives me the 8th character "e";
so I really need to change the number to 5 so I get the 6th character which is "o". 
1st character = position 0 = A 
2nd character = position 1 = v
3rd character = position 2 = a 
4th character = position 3 = _
5th character = position 4 = L 
6th character = position 5 = o  - winner!
7th character = position 6 = v
8th character = position 7 = e

**Comment by andrewek** [comment]
No worries. It all updated appropriately. Thanks. Closing this out because you're done with this one.

**Comment by webbedfooty** [comment]
Hello

 

I’ve noticed that when I hit FORK it doesn’t do that scanner thingy GIF

and then when I go to Cloud 9 to work on the assignment, I am prompted with Creating a Workspace

am I missing a step or doing something out of order?

 

Curiously yours

 

Patrick

**Pull request closed without merging**